src/shared/index.ts

```diff
@@ -0,0 +1 @@
+export * from './exceptions';
```

Comment:
can you delete this or put every other export here

Reply:
I can delete if others don't use it